rustls/tests/unbuffered.rs

```diff
@@ -30,9 +30,6 @@ fn tls12_handshake() {
             "Ok(EncodeTlsData)",
             "Ok(TransmitTlsData)",
             "Ok(BlockedHandshake)",
-            "Ok(BlockedHandshake)",
-            "Ok(BlockedHandshake)",
-            "Ok(BlockedHandshake)",
```

Comment:
it's interesting to observe the optimization so directly in this API's expected states :+1: